client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/iteration/RemoteCloseableIterator.java

```diff
@@ -114,7 +114,7 @@ private void restartIteration(Set<Integer> missedSegments) {
    private void start(Set<Integer> fromSegments) {
```

Comment:
Yes, this can be removed, it's probably a left over from early versions. The same for ```restartIteration```